contrib/cmd/fs-idmap/fs-idmap.go

```diff
@@ -31,20 +31,22 @@ func main() {
 	if err := cmd.Start(); err != nil {
 		log.Fatalf("failed to run the helper binary: %v", err)
 	}
+	defer func() { _ = cmd.Wait() }()
 
 	path := fmt.Sprintf("/proc/%d/ns/user", cmd.Process.Pid)
 	var userNsFile *os.File
 	if userNsFile, err = os.Open(path); err != nil {
 		log.Fatalf("unable to get user ns file descriptor: %v", err)
-		return
```

Comment:
This is an error path, you need to exit here -- otherwise you'll get errors or crashes later when you try to operate on the file. If you're worried about the `Kill`, you should move it to the `defer func() { _ = cmd.Wait() }()` statement.